app/components/date-nav.js

```diff
@@ -0,0 +1,35 @@
+import classic from 'ember-classic-decorator';
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import moment from 'moment';
+
+@classic
+export default class DateNavComponent extends Component {
+  tagName = 'section';
```

Comment:
I think this needs to be the `@tagName` decorator on the class. It might be worth converting the component to a glimmer component?